### DISCUSSION TITLE
May I safely install amiberry-v5.1-rpi4-dmx-32bit-rpios.zip on RetroPie?

### DISCUSSION THREAD

**Discussion by Yserra** [BlitterStudio/amiberry#952] [Q&A] [answered]
Hello all, 
Before breaking everything, I wanted to know if it was ok to simply install the content of amiberry-v5.1-rpi4-dmx-32bit-rpios.zip on RetroPie 4.8/Raspberry Pi 4B, to update what I previously installed from amiberry-v5.0-rpi4-dmx-32bit-retropie.zip, following the instructions from [https://github.com/midwan/amiberry/wiki/Upgrading-to-the-latest-version](https://github.com/midwan/amiberry/wiki/Upgrading-to-the-latest-version)?

If the answer is "no", may I ask when a dedicated Retropie version will be available?

**Comment by midwan** [comment, marked as answer]
@Yserra 
RetroPie has finally updated their script to install Amiberry 5.1, so you shouldn't have to do anything manually anymore. :)
